### ISSUE TITLE
Add Jasmine framework support

### ISSUE THREAD

**Issue by printminion** [winkingzhang/idea-run-wdio#13]
Currently test execution starts with `--framework Mocha`

It would be nice to be able to start tests with `--framework Jasmine`

**Comment by wenqingzhang** [comment]
yup, thanks for good idea, I am still working on this, there are two steps in my minds;

1. can customize (select) framework in configuration UI    .  .  .      _<-  working on here, be ready in next release_
2. automatically detect installed framework and apply it to configuration
